static/src/javascripts/projects/commercial/modules/third-party-tags.spec.js

```diff
@@ -198,22 +207,31 @@ describe('third party tags', () => {
         });
 
         it('should only add consented custom vendors to the document for TCFv2', () => {
-            _.reset();
             shouldUseSourcepointCmp.mockImplementation(() => true);
             onConsentChange.mockImplementation(tcfv2WithConsentMock);
             insertScripts(
-                [fakeThirdPartyAdvertisingTag, fakeThirdPartyPerformanceTag],
+                [fakeThirdPartyAdvertisingTag, fakeThirdPartyAdvertisingTag2],
                 []
             );
             expect(document.scripts.length).toBe(2);
         });
 
+        it('should not add already loaded tags ', () => {
+            fakeThirdPartyAdvertisingTag.loaded = true;
+            shouldUseSourcepointCmp.mockImplementation(() => true);
+            onConsentChange.mockImplementation(tcfv2WithConsentMock);
+            insertScripts(
+                [fakeThirdPartyAdvertisingTag, fakeThirdPartyAdvertisingTag2],
+                []
+            );
+            expect(document.scripts.length).toBe(1);
```

Comment:
I think you don't want to set `loaded` to true and rather call `insertScripts` twice, as this is the real behaviour to test.

Reply:
👍 Changed the test to call the function twice instead of setting loaded to true